components/mpas-seaice/bld/namelist_files/namelist_defaults_mpassi.xml

```diff
@@ -395,11 +395,7 @@
 <config_calc_surface_temperature>true</config_calc_surface_temperature>
 <config_use_form_drag>false</config_use_form_drag>
 <config_use_high_frequency_coupling>true</config_use_high_frequency_coupling>
-<config_use_high_frequency_coupling ice_grid="ECwISC30to60E2r1">false</config_use_high_frequency_coupling>
-<config_use_high_frequency_coupling ice_grid="EC30to60E2r2">false</config_use_high_frequency_coupling>
```

Comment:
>Didn't @jonbob say that the defaults set in Registry.xml are only used for MPAS-SI stand-alone configurations?

@eclare108213 the changes here are what E3SM uses.

Reply:
And so does removing these lines and not replacing them with 'true' mean that this option must always be set in a namelist file?  Isn't that kind of dangerous?

Reply:
L397 sets the option as true by default for all grids. Previously L398,399 changed the default to false only for the `ECwISC30to60E2r1`, `EC30to60E2r2` grids.

Reply:
Ah, sorry I missed that.